sdk/keyvault/keyvault-keys/recordings/browsers/keys_client__create_read_update_and_delete_operations/recording_can_update_key.json

```diff
@@ -1,115 +1,115 @@
+        "x-content-type-options": "nosniff",
+        "x-ms-client-request-id": "4a07acb7-42e8-4249-a7b1-6b26cf0e7f54",
+        "x-ms-keyvault-network-info": "conn_type=Ipv4;addr=50.35.231.105;act_addr_fam=InterNetwork;",
+        "x-ms-keyvault-region": "westus2",
+        "x-ms-keyvault-service-version": "1.9.48.0",
+        "x-ms-request-id": "08ef70a0-3cc0-4723-baef-ffa73a14825c",
+        "x-powered-by": "ASP.NET"
+      }
+    },
+    {
+      "method": "POST",
+      "url": "https://login.microsoftonline.com/12345678-1234-1234-1234-123456789012/oauth2/v2.0/token",
+      "query": {},
+      "requestBody": "response_type=token&grant_type=client_credentials&client_id=azure_client_id&client_secret=azure_client_secret&scope=https%3A%2F%2Fsanitized%2F",
+      "status": 200,
+      "response": "{\"token_type\":\"Bearer\",\"expires_in\":86399,\"ext_expires_in\":86399,\"access_token\":\"access_token\"}",
+      "responseHeaders": {
+        "cache-control": "no-store, no-cache",
+        "content-length": "1310",
+        "content-type": "application/json; charset=utf-8",
+        "date": "Tue, 03 Aug 2021 17:37:27 GMT",
+        "expires": "-1",
+        "nel": "{\"report_to\":\"network-errors\",\"max_age\":86400,\"success_fraction\":0.001,\"failure_fraction\":1.0}",
+        "p3p": "CP=\"DSP CUR OTPi IND OTRi ONL FIN\"",
+        "pragma": "no-cache",
+        "referrer-policy": "strict-origin-when-cross-origin",
+        "report-to": "{\"group\":\"network-errors\",\"max_age\":86400,\"endpoints\":[{\"url\":\"https://identity.nel.measure.office.net/api/report?catId=GW+estsfd+san\"}]}",
+        "strict-transport-security": "max-age=31536000; includeSubDomains",
+        "x-content-type-options": "nosniff",
+        "x-ms-ests-server": "2.1.11898.12 - NCUS ProdSlices",
+        "x-ms-request-id": "3ba8a4ee-1168-4174-9f9f-5c83b832e900"
+      }
+    },
+    {
+      "method": "POST",
+      "url": "https://keyvault_name.vault.azure.net/keys/CRUDKeyName-canupdatekey-/create",
+      "query": {
+        "api-version": "7.3"
+      },
+      "requestBody": "{\"kty\":\"RSA\"}",
+      "status": 200,
+      "response": "{\"key\":{\"kid\":\"https://keyvault_name.vault.azure.net/keys/CRUDKeyName-canupdatekey-/e438663090a24db9bc61eef6c2aa8f35\",\"kty\":\"RSA\",\"key_ops\":[\"encrypt\",\"decrypt\",\"sign\",\"verify\",\"wrapKey\",\"unwrapKey\"],\"n\":\"1qqXH3HFRj_pFrdyn-Rr5029CfHFJz4fKMdTgVMAnq-zeZpip1QdBWwm32n6gkqpV8D5o0arSsPQLcHT1DDk9xNk86EaB-EBHQEn9RNPst7RqSPRScD5LIWg_1zjBVOWYXKDtm4Dcy0rJ9hSslAwKSjUe7YGP9WgLNzzWcNWHBH9UkE4r5s-tDyYpUwduH73sTAkiyb6hxyMff6dx4h61YBBidco96q_TIGUYwRwz5OHmxSf8cTR_by4uUC-ZTbPaB4KO0IQCPSQn9jmMzuZCbm_dS1ckbKYl-0FI7-45G1pOucNhUjytiWntjwbHpGfXgaXZPwncnbwAuHqu4tuEQ\",\"e\":\"AQAB\"},\"attributes\":{\"enabled\":true,\"created\":1628012247,\"updated\":1628012247,\"recoveryLevel\":\"CustomizedRecoverable+Purgeable\",\"recoverableDays\":7}}",
```

Comment:
is the response value here sanitized or safe? (just confirming)

Reply:
This and the csr question are good checks but not necessary to sanitize. Every key is created for the test and destroyed at the end of the tests so there's nothing here that is unsafe